### ISSUE TITLE
Wrap "Child records" in li on record page

### ISSUE THREAD

**Issue by eliotjordan** [pulibrary/pulmap#422]
Referenced in #412

**Comment by eliotjordan** [comment]
Addressed in upstream PR: https://github.com/geoblacklight/geoblacklight/pull/571
